### DISCUSSION TITLE
How I can specify default `NULL` in table?

### DISCUSSION THREAD

**Discussion by Desiders** [SeaQL/sea-query#837] [Q&A] [answered]
I saw a similar [discussion](https://github.com/SeaQL/sea-orm/discussions/1035) about CURRENT_TIMESTAMP, but how to do it with _NULL_?

**Comment by Desiders** [comment]
Currently, I just use `.extra("DEFAULT NULL")`

**Comment by tyt2y3** [comment]
Ref: https://docs.rs/sea-query/latest/sea_query/table/struct.ColumnDef.html#method.default

**Reply by Desiders** [reply]
> Ref: https://docs.rs/sea-query/latest/sea_query/table/struct.ColumnDef.html#method.default

Thanks for the quick reply. I have seen this, but I don't know how to specify `NULL` in this method.
I found an option only in the form of `.default(SimpleExpr::Custom("NULL".to_owned()))`

**Reply by Desiders** [reply]
> I found an option only in the form of `.default(SimpleExpr::Custom("NULL".to_owned()))`

There is also an option  `Expr::new_with_left(Keyword::Null)`, but it's a private API

**Reply by Desiders** [reply, marked as answer]
> There is also an option `Expr::new_with_left(Keyword::Null)`, but it's a private API

I think we need a method like `Expr::null()`, analogous to `Expr::current_timestamp()`.
If you agree, then I can create a PR.
